File: python/is_a_number_prime.py
# https://www.codewars.com/kata/5262119038c0985a5b00029f
def is_prime(num):
    if num <= 1:
        return False
    
    for i in range(2, int(num**0.5) + 1):
        if num % i == 0:
            return False
    
    return True

File: go/is_a_number_prime.go
// https://www.codewars.com/kata/5262119038c0985a5b00029f
package kata

import "math"

func IsPrime(n int) bool {
  //your code here
  if n <= 1 {
    return false
  }
  
  m := int(math.Sqrt(float64(n)))
  
  for i := 2; i < m + 1; i++ {
    if n % i == 0 {
      return false
    }
  }
  return true
}
